agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -116,6 +116,11 @@ function collect_sos {
 		# ref for plugin tuned: https://github.com/sosreport/sos/commit/c55c58cf2dabf93c924c839c8ed045c18e31ba1c
 		_modules="${_modules} tuned"
 	fi
+        #OpenStack plugin
+        OSP=/etc/rhosp-release
+        if [[ -f "$OSP" ]] ; then
+                _modules="${_modules} openstack_ansible openstack_aodh openstack_ceilometer openstack_cinder openstack_glance openstack_heat openstack_horizon openstack_instack openstack_ironic openstack_keystone openstack_manila openstack_neutron openstack_nova openstack_novajoin openstack_octavia openstack_placement openstack_swift os_net_config mssql cman ceph openvswitch ovn_central ovn_host pacemaker podman rabbitmq kernelrt ceph ceph-ansible kvm corosync"
```

Comment:
@pradiptapks, is `ceph` supposed to be in the list twice?

@dbutenhof, should we be considering doing duplicate-removals on this list, or does it actually matter (i.e., will sosreport handle that adequately)?

Reply:
We can't do plugins this way, because they are very tied to the version of the OS.  Each one has to be check against the versions of `sosreport` that we execute against (RHEL 7.7, .8, .9, RHEL 8.*, RHEL 9, Fedora 32 & 33, CentOS 7 & 8, CentOS Streams?).

Reply:
@webbnh I removed `ceph_ansible` as `ceph` is sufficient to collect the log information.

@portante OpenStack parsers in `sosreport` are common to RHEL7.x, 8.x, CentOS, Fedora. As `podman` has introduced in RHEL8.x, I wonder whether it does backwards support below version 3. May I know which version of so start support podman, so that I can change the condition.